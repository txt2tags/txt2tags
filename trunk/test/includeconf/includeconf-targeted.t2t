
%!target: html
%!includeconf(html): _failed.inc

= title =
