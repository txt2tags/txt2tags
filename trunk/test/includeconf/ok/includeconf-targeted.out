txt2tags: Error: You can't use (target) with %!includeconf
%!includeconf(html): _failed.inc
